File: descriptive/univariate/stations/stations_analysis.py
import pandas as pd

# Load the data from the pickle file
stations_data = pd.read_pickle("stations_data.pkl")

# Display the first few rows of the data
print("First 3 entries:")
print(stations_data.head(3))

# Display summary statistics for numeric columns
print("\nSummary statistics for numeric columns:")
print(stations_data.describe())

# Display the data types of each column
print("\nData types of each column:")
print(stations_data.dtypes)

File: descriptive/stations/stations_analysis.R
library(ggplot2)

# Load the data
stations_data <- readRDS("stations_data.rds")

# Summary of the data
summary_data <- summary(stations_data)
print(summary_data)

# Boxplot for numeric columns
numeric_columns <- sapply(stations_data, is.numeric)
boxplot(stations_data[, numeric_columns], main = "Boxplot of Numeric Columns")

